src/wallet/wallet.cpp

```diff
@@ -3293,6 +3296,12 @@ std::string CWallet::GetWalletHelpString(bool showDebug)
 
 bool CWallet::InitLoadWallet()
 {
+    if (GetBoolArg("-disablewallet", false)) {
```

Comment:
I wonder why we don't have a constant DEFAULT_DISABLEWALLET like for the other args.